### DISCUSSION TITLE
Pinned threads to replace FAQs

### DISCUSSION THREAD

**Discussion by ara4n** [element-hq/element-meta#1304] [Ideas]

**Comment by ara4n** [comment]
I spend my life having the same conversations on HQ: explaining matrix v xmpp, http v websockets, whether matrix is funded, why e2e doesn't work yet, etc. It feels rude and patronising to tell people to RTFM or the FAQ but a really nice solution when we have threading might be to maintain an "official thread" for these discussions which people can be pointed to. It's like saying "oh, we were just discussing that the other day! here's the permalink!" but with threading, and hopefully avoids redundancy without being rude.

**Comment by lampholder** [comment]
100% agreed, but it's a shame this is blocked behind threading. Is there any small optimisation we can make in the meantime?

**Comment by lampholder** [comment]
https://github.com/vector-im/riot-web/issues/2349

**Comment by MurzNN** [comment]
Maybe good first step will be implement FAQ bot that listen messages, waiting matching to FAQ rules message and post FAQ posts about them? For example, on message with words "matrix, xmpp, difference" - post FAQ answer about this.
Because even if we implement Pinned threads, users will not look at it and read, they continue asking simple things many times :)

**Comment by MurzNN** [comment]
And at first - FAQ treads may be separate Matrix rooms, where the name of room is question, topic is answer, messages are comments. For example, room "Matrix vs XMPP difference".
And we can integrate this rooms via Matrix Static like https://view.matrix.org/room/!cURbafjkfsMDVwdRDQ:matrix.org/ in https://matrix.org/docs/guides/faq.html page

**Comment by ara4n** [comment]
Heads up that pinning was actually contributed and merged to develop a while back by @turt2live.  It has a few remaining bugs which keep it hidden behind a Labs flag, but it's almost there.
https://github.com/vector-im/riot-web/pull/5142

**Comment by daniellekirkwood** [comment]
_Moving to Discussions as is the process_

Personally I think this is a great use of threads. For the moment we're not actively working on threads V2 features and therefore this is a discussion, not an issue :)
